compiler/ballerina-parser/src/test/resources/types/tuple-type/tuple_type_source_10.bal

```diff
@@ -0,0 +1,7 @@
+type T1 [@annot {} int, @annot {} string];
```

Comment:
To cover the optional mapping-constructor expr in annotation and a type ref scenario
```suggestion
type T1 [@annot {} int, @annot {} string, @annot T];
```